setup/terraform/resources/setup-ecs.sh

```diff
@@ -35,6 +35,17 @@ if [[ $ACTION == "install-prereqs" ]]; then
   fi
 
   deploy_os_prereqs
+  # Install dependencies on python38 for ECS 1.5.2
```

Comment:
This is the same set of steps used to install Python 3.8 on the cluster node, right?
Could you make this into a "install_python" function in common.sh and use that same function on both setup.sh and setup-ecs.sh?

It's going to be cleaner and easier to maintain.